tasks/Task.cpp

```diff
@@ -845,6 +845,12 @@ void Task::exposureConfiguration(Arena::IDevice& device)
         }
     }
     else if (image_config.exposure_auto == ExposureAuto::EXPOSURE_AUTO_OFF) {
+        double exposure_time_seconds = image_config.exposure_time.toSeconds();
+        double seconds_per_frame = 1 / image_config.frame_rate;
+        if (exposure_time_seconds > seconds_per_frame + 0.0001) {
```

Comment:
Actually ... if anything, the tolerance should be going the opposite way. I would guess that one needs the exposure to be slightly smaller than the seconds per frame, not longer.